Makefile

```diff
@@ -140,6 +140,8 @@ uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified
 	$(KUSTOMIZE) build config/crd | kubectl delete -f -
 
 deploy: gen-manifests ## Deploy controller to the K8s cluster specified in ~/.kube/config.
+	kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.7.1/cert-manager.yaml
```

Comment:
pls make sure we revert the changes in `make undeploy`

Reply:
done